ingest_munkholmen_pi_status.py

```diff
@@ -19,7 +19,7 @@ def main():
     admin_user, admin_pwd = util_file.get_user_pwd(os.path.join(config.secrets_dir, 'influx_admin_credentials'))
     clients = [
         InfluxDBClient(config.az_influx_pc, 8086, admin_user, admin_pwd, 'oceanlab'),
-        InfluxDBClient(config.sintef_influx_pc, 8086, admin_user, admin_pwd, 'test'),
+        # InfluxDBClient(config.sintef_influx_pc, 8086, admin_user, admin_pwd, 'test'),
```

Comment:
should we delete this instead of commenting out?

Reply:
Done.

There are of course still larger problems here, and now we are hiding those problems better, but that's not really a part of this PR so I'm happy to ignore for now. I think I'll just make an 'issue' that we aren't always treating the two DBs the same.